packages/cli/src/commands/start.ts

```diff
@@ -197,6 +197,10 @@ export class Start extends BaseCommand {
 			}
 		}
 
+		if (process.env.OFFLOAD_MANUAL_EXECUTIONS_TO_WORKERS === 'true') {
+			this.needsTaskRunner = false;
+		}
+
```

Comment:
Should add a test for this? There's a very similar one at `packages/cli/src/commands/__tests__/execute.test.ts`

Reply:
Gave it a try but as usual commands are painful to test, so this little change is not worth the time. Let's disentangle this command some other time.